wrapanapi/systems/virtualcenter.py

```diff
@@ -946,6 +946,13 @@ def list_cluster(self):
     def list_resource_pools(self):
         return [str(h.name) for h in self.get_obj_list(vim.ResourcePool)]
 
+    def list_networks(self):
+        """Fetch the list of network objects
+
+        Returns: A list of vim.Network objects
```

Comment:
docblock is inaccurate here, this is returning a list of network names.

Reply:
will fix this, thanks